pkg/gcptarget/library.go

```diff
@@ -59,37 +59,6 @@ autoreject_review[rejection] {
 path_matches(path, pattern) {
 	glob.match(pattern, ["/"], path)
 }
-
-########
```

Comment:
It's no longer necessary because `get_default` is now implemented in rego core as `object.get`.

I initially removed this because I was having trouble reasoning about where a bug in the various rego files was coming from; I could revert it for now, though I do think it's something we should do in the long run.

Reply:
This would break all existing templates, so we definitely can't remove it.

Feel free to not include it in the tftarget though.

Reply:
I don't think this is made available to existing templates - don't they get `get_default` from the policy-library lib directory? https://github.com/GoogleCloudPlatform/policy-library/blob/master/lib/util.rego

Reply:
Passing the library in from the `policy-library` directory is legacy behavior that we never resolved fully. Constraint Framework is supposed to allow Targets to provide a set of libs that templates can consume (ex. https://github.com/open-policy-agent/gatekeeper/tree/b36e85531dbe9b4abfccf86dd9746bbe2f98e004/pkg/target/regolib).

Reply:
Yes, we did implement it: https://github.com/GoogleCloudPlatform/config-validator/blob/master/pkg/gcptarget/gcptarget.go#L80

We should probably work on removing the `lib/` directory requirement from TFV.

Reply:
+1 on removing the `lib` directory requirement from TFV.

I'll go ahead and revert this removal for now. I'm still not clear how someone would have actually referenced this library from constraint templates or that anyone actually did so, since I haven't seen any documentation on how to use it, and all the policy-library templates use `data.validator.gcp.lib` - but it's not critical for this PR.